docs/source/deploy.rst

```diff
@@ -0,0 +1,135 @@
+======
+Deploy
+======
+
+The deployment docs are split up in two parts. First there is the
+general section, which should always be followed. Then there is a cloud
+service provider specific section of which one provide should be chosen.
+
+Setup an example project
+========================
+
+1. Create a project directory of notebooks you wish to display. For this
+   tutorial we will clone voila and treat the notebooks folder as our
+   project root.
+
+   .. code:: bash
+
+       $ git clone git@github.com:QuantStack/voila.git
+       $ cd voila/notebooks/
+
+2. Add a requirements.txt file to the project directory. For this
```

Comment:
After:

> Add a `requirements.txt` file to the project directory

it seems useful to add a sentence that explains the point of this file. Something like "This file should contain all the Python dependencies your voila app needs to run."

Reply:
Added your suggestion